### ISSUE TITLE
Multi-box loss dimensions

### ISSUE THREAD

**Issue by oarriaga** [rykov8/ssd_keras#43]
Hello @rykov8 ,

In the file ssd_utils.py in the assign_boxes method you mention:
> assignment[:, -7:] are all 0. See loss for more details

 I believe that `assignment[:, -8:]` counts the number of positive examples  (examples assigned to a prior box); however, I have not find any use of  all the zeros between `assignment[:, -7:-1]` in the loss function. 
Did I miss something in regards to their use? or should we change the loss function so that it contains only the counter at the end?
If so, we could probably substitute the counter from `y_true[:, :, -8]`, with the assigned probability values for the ground truth bounding boxes for the background class in `y_true[:, :, 4]` right? 

Thanks

**Comment by rykov8** [comment]
@oarriaga Actually, these auxiliary zeros are needed for `Keras`. It cannot handle `y_true` and `y_pred` of different sizes, so, even if in `y_true` there is no need to provide priors coordinates, I have to preserve the shape. So, these zeros are not used, but are added for `Keras` compatability.

As for the second part of your question, probably, you're right. If we classify some prior as background prior, we can say, that is was not assigned as a ground truth prior. If you want to do this, you'll also need to change `MultiboxLoss`. But in any case, you need to output auxiliary zeros at last columns instead of priors for the reasons described above.
